src/code/z_actor.c

```diff
@@ -687,18 +687,19 @@ void Target_Update(TargetContext* targetCtx, Player* player, Actor* targetedActo
             targetedActor->world.pos.y - (targetedActor->shape.yOffset * targetedActor->scale.y);
         targetCtx->targetCenterPos.z = targetedActor->world.pos.z;
 
-        if (targetCtx->rotation == 0) {
+        if (targetCtx->rotZIndex == 0) {
             f32 temp_f0_2 = (500.0f - targetCtx->unk44) * 3.0f;
             f32 clampedFloat;
 
             clampedFloat = CLAMP(temp_f0_2, 30.0f, 100.0f);
 
             if (Math_StepToF(&targetCtx->unk44, 80.0f, clampedFloat)) {
-                targetCtx->rotation++;
+                targetCtx->rotZIndex++;
             }
         } else {
             // 0x80 is or'd to avoid getting this value be set to zero
-            targetCtx->rotation = (targetCtx->rotation + 3) | 0x80;
+            // This rotation value gets multiplied by 0x200, which multiplied by 0x80 gives a full turn (0x10000)
+            targetCtx->rotZIndex = (targetCtx->rotZIndex + 3) | 0x80;
```

Comment:
I think it's still worth mentioning that it's 'or'd to avoid being set to 0, and that it just uses a full turn to not affect the calculation

Reply:
The comment about 0 is still there, I think github messed up a diff view